dracut/30ignition/ignition-disks.service

```diff
@@ -2,13 +2,17 @@
 Description=Ignition (disks)
 DefaultDependencies=false
 
-Requires=sysinit.target
-After=sysinit.target
+# This stage runs between `basic.target` and `initrd-root-fs.target`,
+# see https://www.freedesktop.org/software/systemd/man/bootup.html
+After=basic.target
+Before=initrd-root-fs.target
 
-Requires=network.target
+# Network may be used to fetch userdata content.
 After=network.target
```

Comment:
no need for `Requires=network.target` ?

Reply:
That's a tricky topic. We don't strictly require networking (eg. libvirt with no additional remote URLs), but if networking is going to be available we prefer to be started after that. I will revert back to the original unit, as it seems less surprise prone.

Reply:
yeah I think `network.target` is a bit confusing anyway. 

from https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/:

- *network.target has very little meaning during start-up. It only indicates that the network management stack is up after it has been reached.*
- *network-online.target is a target that actively waits until the nework is "up"*